gix-date/src/lib.rs

```diff
@@ -29,6 +32,24 @@ pub struct Time {
     pub sign: time::Sign,
 }
 
+impl Sub for Time {
+    type Output = i64;
+
+    fn sub(self, rhs: Self) -> Self::Output {
+        self.total() - rhs.total()
+    }
+}
+
+impl Time {
+    fn total(&self) -> i64 {
```

Comment:
This wants to be `local_time()` really, as it's the amount of seconds one would see with the offset applied.

Reply:
this implies also needs to be in the time module right?

Reply:
Yes.